lib/models/asset-size-printer.js

```diff
@@ -63,16 +63,16 @@ module.exports = class AssetPrinterSize {
         })
         // Print human-readable file sizes (including gzipped)
         .map((file) => {
-          let contentsBuffer = fs.readFileSync(file);
-          return gzip(contentsBuffer).then((buffer) => ({
-            name: file,
-            size: contentsBuffer.length,
-            gzipSize: buffer.length,
-            showGzipped: contentsBuffer.length > 0,
-          }));
+          return pool
+            .exec('gzipStats', [file])
+            .then((result) => result)
```

Comment:
shouldn't we remove this ?

Reply:
Remove what? @bekzod Can you please elaborate?

Reply:
sorry, I meant ` .then((result) => result)`

Reply:
`.then((something) => something)` is the same as not adding the `.then` at all.